packages/fluentui/react-northstar/src/components/Carousel/Carousel.tsx

```diff
@@ -297,29 +332,29 @@ export const Carousel = (React.forwardRef<HTMLDivElement, CarouselProps>((props,
             items.map((item, index) => {
               const itemRef = itemRefs[index];
               const active = activeIndex === index;
-              let slideToNext = prevActiveIndex < activeIndex;
-              const initialMounting = prevActiveIndex === -1;
+              const initialMounting = typeof dir === 'undefined';
```

Comment:
I suggest to move this out as a function and cover it with unit tests.

Reply:
You mean the `renderContent`?

Reply:
I mean the code that computes animation name for an item.